app/_hub/kong-inc/request-validator/_index.md

```diff
@@ -463,12 +463,15 @@ In this example, use the plugin to validate a request's path parameter.
     }
     ```
 
-4. In these step examples, validation makes sure that `status_code` is a number.
+4. In these step examples, validation makes sure that `status_code` is a number and that the body contains a parameter called `name`
 
    A proxy request with a non-numerical status code is blocked:
 
     ```
-    curl -i -X GET http://kong:8000/status/abc
+    curl --request POST \
```

Comment:
```suggestion
    curl -i -X POST \
```
Using the shorthand for a request + headers for consistency with the rest of the docs site.